### PR TITLE
Comment table

### PR DESCRIPTION
**Pull request by patofernandez**
image

### CODE REVIEW

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { View, Image, Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class CommentItem extends Component {
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: replace this class with an arrow function component:
```
const CommentItem = () => (
...
);
```

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/styles.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	container: {
+        alignItems: 'flex-start',
+        flexDirection: 'row',
+        backgroundColor: 'white',
+        borderRadius: 5,
+        padding: 15
+    },
+    userImage: {
+        width: 45,
+        height: 45,
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/styles.js]: extract common size in a constant value in this file

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+		return (
+      <View style={styles.commentTable}>
+        <ScrollView>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+          <CommentItem/>
+        </ScrollView>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: we should be thinking that you will receive a variable comment's list size. Replace this with a FlatList

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import {  
+  Text,
+  View,
+  ScrollView
+} from 'react-native';
+
+import CommentItem from './component/CommentItem';
+import styles from './styles';
+
+class CommentTable extends Component {
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: turn this into an arrow function

File: wbooks/src/app/screens/BookDetail/index.js [hunk 5]
```diff
@@ -35,7 +36,9 @@ class BookDetail extends Component {
             </TouchableOpacity>
           </View>
         </View>
-        <View style={styles.cardDetail} />
+        <View style={styles.cardDetail}>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/index.js]: let's talk about this later

Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js (hunk 1 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/styles.js (hunk 2 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/styles.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js (hunk 3 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js (hunk 4 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: do this is no longer necessary

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { View, Image, Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+const CommentItem = ({ userName, comment, uri }) => (
+  <View style={styles.container}>
+    <Image style={styles.userImage}
+    source={{uri:"http://wolox-training.s3.amazonaws.com/uploads/el-psicoanalista-analyst-john-katzenbach-paperback-cover-art.jpg"}}/>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: use the uri

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    <View style={styles.textContainer}>
+      <Text style={styles.userName}>John Appleseed</Text>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: use the username

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+import styles from './styles';
+
+const CommentItem = ({ userName, comment, uri }) => (
+  <View style={styles.container}>
+    <Image style={styles.userImage}
+    source={{uri:"http://wolox-training.s3.amazonaws.com/uploads/el-psicoanalista-analyst-john-katzenbach-paperback-cover-art.jpg"}}/>
+    <View style={styles.textContainer}>
+      <Text style={styles.userName}>John Appleseed</Text>    
+      <Text style={styles.comment}>It was great to see you again earlier. Let’s definitely get that coffe It was great to see you again earlier.</Text>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: use the comment

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+import CommentItem from './component/CommentItem';
+import styles from './styles';
+import COMMENTS from '../../comments';
+
+class CommentTable extends Component {
+
+  renderItem = ({ item }) => (
+    <TouchableOpacity 
+      onPress={() => {}}
+    >
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: i think we won't need the comment to be touchable

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js (hunk 6 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js (hunk 7 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js (hunk 8 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js (hunk 9 above)
Comment (patofernandez) [reply wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: done

File: wbooks/src/app/screens/BookDetail/components/CommentTable/index.js [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+	render() {
+
+		return (
+      <View style={styles.commentTable}>
+        <FlatList
+          data={COMMENTS}
+          renderItem={this.renderItem}
+          keyExtractor={this.keyExtractor}
+        />
+        {COMMENTS.length > 5 && (<Text style={styles.linkText}>View All</Text>)}
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/CommentTable/index.js]: i think this should be a button

**Pull request merged**